### PR TITLE
Traces/Vitamins/Nucleotides/Minerals/Other tags

### PR DESCRIPTION
**Pull request by aleene**
## PR Description

Shows Minerals, etc as a separate line.

Type of Changes 

- [ ] Fixes Issue #212

### CODE REVIEW

**Comment by teolemon** [comment]
Is it mergeable ? Otherwise can we release it without it ?

**Comment by aleene** [comment]
This one can be merged. Did it a long time ago. Was for the minerals, nucleotides, etc.

File: Sources/ViewControllers/Products/Detail/ProductDetailViewController.swift [hunk 1]
```diff
@@ -184,8 +184,53 @@ class ProductDetailViewController: ButtonBarPagerTabStripViewController, DataMan
                 }
             }
         }
+        createFormRow(with: &rows, item: product.traces?.map({ (trace: Tag) -> NSAttributedString in
```
Comment (teolemon) [inline Sources/ViewControllers/Products/Detail/ProductDetailViewController.swift]: Can it be factorized ?

File: Sources/Network/TaxonomiesService.swift [hunk 2]
```diff
@@ -102,6 +106,88 @@ class TaxonomiesService: TaxonomiesApi {
         }
     }
 
+    fileprivate func refreshVitamins(_ callback: @escaping (_: Bool) -> Void) {
```
Comment (teolemon) [inline Sources/Network/TaxonomiesService.swift]: Can it be factorized ?

**Comment by teolemon** [comment]
also 3 tiny merge conflicts

**Pull request merged**